README.md

```diff
@@ -1,30 +1,56 @@
-# Front-End Repo for Node/React Practicum
+# JSQuiz App
 
-This will be the front-end for your team's practicum project.
+JSQuiz is for learning developers to test their knowledge and understanding of coding skills. Users can create an account and take quizzes on many different topics with varying difficulty levels. Take notes on quizzes and revisit them by going to your profile.
 
-It is suggested that you run these instructions **after** you setup the back-end server first.
-You can go through these steps during your first group meeting in case you need assistance from your mentors.
+## Table of Contents
 
-You will have two folders inside one team folder (one for front-end and one for back-end). Name the parent folder something appropriate (in the below example we title it "Practicum Project"). Then clone directly (do not fork and clone) the front and back repos while inside the parent ("Practicum Project") project folder.
+- [Installation](#installation)
+- [Usage](#usage)
+- [Credits](#credits)
 
-![folders](images/folder_structure.png)
+## Installation
 
-> The front-end app (React) will be running on port 3000. The back-end server will be running on port 8000. You will need to run both the front-end app and the back-end server at the same time to test your app.
+To run JSQuiz on your local machine, follow these steps for installation:
 
-### Setting up local development environment
+Backend:
 
-1. Clone this repository to the folder that was already created for both the front-end and back-end repos
+1. Clone the repository: `https://github.com/Code-the-Dream-School/ee-prac-team3-back`
+2. Run `npm install` to install dependencies
```

Comment:
I believe the backend needs a `.env` file created. Bino added some instructions here: https://codethedream.slack.com/archives/C061FVBGZU0/p1701554621350719

Reply:
Hi Elijah, I deleted my configuration part in the previous commit because I thought this was meant to be private and not shared. Is that true or should I add this back in? Or something else?

```
 ## Configuration

JSQuiz can be configured using environment variables. Create a `.env` file in the root directory with the following:

PORT=8000
MONGO_URI=mongodb+srv://alinaM:3azzPHIVVhHN0sn0@cluster0.ki4rdbc.mongodb.net/Quizzes
SECRET=SECRET
CLIENT_URL=http://localhost:3000
```